src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/UpdateGoalTest.kt

```diff
@@ -180,6 +180,9 @@ class UpdateGoalTest : IntegrationTestBase() {
         aValidCreateStepRequest(
           title = "Book course",
         ),
+        aValidCreateStepRequest(
```

Comment:
I've added a new step in the original Goal in this Update Goal integration test.
The POST request does not include this goal, so it is seen as a deleted step, so therefore we can assert the tracking event is sent 👍